build-logic/buildquality/src/main/kotlin/gradlebuild.arch-test.gradle.kts

```
@@ -61,7 +61,7 @@ testing {
                         testClassesDirs += sharedArchTestClasses.filter { it.isDirectory }
                         classpath += sourceSets.main.get().output.classesDirs
                         systemProperty("package.cycle.exclude.patterns", packageCyclesExtension.excludePatterns.get().joinToString(","))
-                        configure<PredictiveTestSelectionExtension> {
+                        extensions.findByType<PredictiveTestSelectionExtension>().apply {
```

Shouldn't this be made null safe?
```suggestion
                        extensions.findByType<PredictiveTestSelectionExtension>()?.apply {
```

---

Yes, you are right!